src/v/cluster/feature_manager.cc

```diff
@@ -397,7 +397,7 @@ ss::future<> feature_manager::do_maybe_update_active_version() {
 
 ss::future<std::error_code>
 feature_manager::update_license(security::license&& license) {
-    static const auto timeout = model::timeout_clock::now() + 5s;
+    const auto timeout = model::timeout_clock::now() + 5s;
```

Comment:
> - Making this timeout a static var meant that 5s after first function
invocation a 0 timeout would be passed to `replicate_and_wait`

nice find. and another reason why we should be transitioning timeouts to use durations when possible.